.github/workflows/pr-build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     name: Test
     needs: build
     runs-on: ubuntu-latest
-    timeout-minutes: 90
+    timeout-minutes: 200
```

Comment:
Not sure if changes in this file should be merged, please check with devops

Reply:
these changes were made by me

its okey...